src/api/src/logger/WinstonLogger.ts

```diff
@@ -93,4 +93,11 @@ export default class WinstonLoggerService implements LoggerService {
 
     return this.logger.verbose(message, { context });
   }
+
+  public query(
+    options?: QueryOptions | undefined,
```

Comment:
It should either function params made from basic types or an interface imported from the generic logger interface.